iina/HistoryWindowController.swift

```diff
@@ -95,22 +95,32 @@ class HistoryWindowController: NSWindowController, NSOutlineViewDelegate, NSOutl
     historyData[key]!.append(entry)
   }
 
+  private func removeAfterConfirmation(_ entries: [PlaybackHistory]) {
+    Utility.quickAskPanel("delete_history", sheetWindow: window) { respond in
+      guard respond == .alertFirstButtonReturn else { return }
+      HistoryController.shared.remove(entries)
+    }
+  }
+
   // MARK: Key event
 
   override func keyDown(with event: NSEvent) {
-    let commandKey = NSEvent.ModifierFlags.command
-    if event.modifierFlags.intersection(.deviceIndependentFlagsMask) == commandKey  {
+    let flags = event.modifierFlags.intersection(.deviceIndependentFlagsMask)
+    if flags == .command  {
       switch event.charactersIgnoringModifiers! {
-      case "f":
-        window!.makeFirstResponder(historySearchField)
-      case "a":
-        outlineView.selectAll(nil)
-      default:
-        break
+        case "f":
+          window!.makeFirstResponder(historySearchField)
+        case "a":
+          outlineView.selectAll(nil)
+        default:
+          break
+      }
+    } else {
+      let key = KeyCodeHelper.mpvKeyCode(from: event)
```

Comment:
Using `mpvKeyCode` here is a little weird to me, although its not unacceptable, but don't we have other elegant way to do this?

Reply:
Well - I don't think so....It looks like MacOS's original design was never made more friendly in this regard, so I think we either have to use hexcodes or use our own constants. I was following the pattern set [in the welcome window.](https://github.com/iina/iina/pull/3739#issuecomment-1139288915).

Reply:
Maybe later on we can refactor this part, maybe to abstract another helper function...